oauth/o-auth.md

```diff
@@ -61,29 +61,35 @@ The first thing you want to do is set up your credentials on google by following
 Please make sure that you secure ALL your private keys and sensitive information under your ``.env`` files! </br> 
 #### Step 2 - Connect Google credentials to Auth0 in Connections </br> </br> 
 Log into Auth0, on the left side you will see a menu with a ``Connections`` option. Select ``Connections`` then select the sub category named ``Social`` that will lead you to select the Gmail connection as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png"  width="500" height="500" /></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/connectionstab.png" width="500" height="500" /></br>
 
 Once you click on ``Try it!`` you will see a modal pop up where you will have to input the Google credentials you created earlier. Just as follows: </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png"  width="500" height="500" /></br></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/gmailconnectionmodal.png" width="500" height="500" /></br></br>
 #### Step 3 - Create Auth0 application to download code files </br> 
 Once you've successfully integrated your Google credentials with the connections section of Auth0 you can proceed with going to ``Applications`` and then type in ``React`` in the search bar. </br>
-<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/Screen%20Shot%202018-10-09%20at%205.58.09%20PM.png"  width="500" height="500" /></br></br>
+<img src="https://github.com/mmedina3/Auth0-Lesson/blob/master/Screen%20Shot%202018-10-09%20at%205.58.09%20PM.png" width="500" height="500" /></br></br>
+
+#### Step 4 - Create a working log-out button such that the button is shown when the user is authenticated, providing an option to log-out successfully </br>
+Once the user has successfully login,they are rendered with the option of log-out button, the following code in the *app.js* file can be added to obtain so.</br>
```

Comment:
user has successfully 'logged in'

Reply:
made the following change in my recent commit.